patterns/footer.php

```diff
@@ -1,6 +1,6 @@
 <?php
 /**
- * Title: Default footer
+ * Title: Three column footer with colophon
```

Comment:
When I read these names it makes me think that the editor needs to be updated to show the description by default. It is visually hidden now.

Reply:
Yes, it is always confusing with the name.

Reply:
I think we don't need the word, since when we clear customizations we will be back to that one. In the end the important bit is that the user finds one that they like, it doesn't matter if it is the default one or not.